catalog/configurable-gauge/static/mod-manifest.json

```diff
@@ -0,0 +1,119 @@
+{
+    "apiVersion": "1.3",
+    "version": "1.0",
+    "name": "Gauge",
+    "id": "spotfire-configurable-gauge-mod",
+    "icon": "icon.svg",
+    "properties": [
+        {
+            "name": "gaugeWidth",
+            "defaultValue": 20,
+            "type": "integer"
+        },
+        {
+            "name": "gaugeOpacity",
+            "defaultValue": 100,
+            "type": "integer"
+        },
+        {
+            "name": "gaugeAngle",
+            "defaultValue": 40,
+            "type": "integer"
+        },
+        {
+            "name": "ticksOpacity",
+            "defaultValue": 100,
+            "type": "integer"
+        },
+        {
+            "name": "showPercent",
+            "defaultValue": "false",
+            "type": "boolean"
+        },
+        {
+            "name": "showShake",
```

Comment:
Consider renaming to "vibrate", or similar. Both here and in the UI. I think that would be easier to understand.